Developers/Newcomers_Guide.rst

```diff
@@ -3,8 +3,8 @@
 Welcome to the Newcomers Guide!
 ===============================
 
-To become part of the coala developers team, there's a few steps you need
-to complete. The newcomer process is as it follows:
+To become part of the coala developers' team, there are few steps you need
```

Comment:
Mmm, I think, I don't get your comment :) Basically, "there's a few steps" is wrong grammar in English..